src/test/java/com/github/shyiko/mysql/binlog/BinaryLogClientIntegrationTest.java

```diff
@@ -1140,6 +1140,17 @@ public void execute(final ResultSet rs) throws SQLException {
         });
     }
 
+    @Test
+    public void testMySQL8InvisibleColumn() throws Exception {
```

Comment:
Note this does really trigger the reported issue, i.e. I don't get the new metadata type. So while the test passes, it doesn't seem to cover the originally reported situation yet. Waiting to hear back from @tinntsea on this.

Reply:
Hi Gunnar,

As I remember I could reproduce the issue last time.
Could you try with the following statements:

```
create table user_table_test1 (
    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
    name varchar(100) not null,
    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP INVISIBLE
);

insert into user_table_test1 (name)
values ('User 1')
;
```

Reply:
Hum, doesn't make a difference; don't receive that metadata type :( Really strange.

Reply:
Hi @gunnarmorling ,

Is everything good with this PR? Or anything I can help with?

Reply:
Hi @gunnarmorling

I tested your changes in this PR and it worked with the invisible columns (while master branch does not)

The event looks like:

```
Event{header=EventHeaderV4{timestamp=1623769869000, eventType=TABLE_MAP, serverId=3179378664, headerLength=19, dataLength=294, nextPosition=59604251, flags=0}, data=TableMapEventData{tableId=2976, database='my_database', table='my_table', columnTypes=-2, -2, -2, 15, 15, 15, 8, 3, 18, -4, 18, 18, 16, 15, 15, columnMetadata=65040, 65040, 65040, 128, 512, 128, 0, 0, 0, 2, 0, 0, 1, 1020, 1020, columnNullability={3, 4, 9, 12}, eventMetadata=TableMapEventMetadata{signedness={}, defaultCharset=255, charsetCollations=0: 63, 1: 63, 2: 63, 6: 63, columnCharsets=null, columnNames=id, partner_id, user_id, agency_record_key, list_name, license_plate, offense_category_id, offense_priority, record_exp, other, created_date, modified_date, deleted, registration_state, registration_country, setStrValues=null, enumStrValues=null, geometryTypes=null, simplePrimaryKeys=0, primaryKeysWithPrefix=null, enumAndSetDefaultCharset=null, enumAndSetColumnCharsets=null,visibility={0, 1}}}}
```